code_challenges/hacker_rank/nested_lists.py

```diff
@@ -0,0 +1,32 @@
+def student_grades(records):
+
+    lowest_score = 100
+    second_lowest_score = 100
+    lowest_students = []
+    second_lowest_students = []
+    for student_name_and_score in records:
```

Comment:
Instead of `for student_name_and_score in records:` how about expanding it out so the code below is a bit easyer to read with `for name, score in records:`